### PR TITLE
Built-in rate limiter - 20200303

### PR DESCRIPTION
**Pull request by jamesliupenn**
As part of ENG-1086 & SUP-34

### CODE REVIEW

**Comment by jamesliupenn** [comment]
@dbhoot do you mind taking a look at the changes made and provide some feedback?

File: README.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ Once the Onfleet object is created, you will get access to all the API endpoints
 Run `npm test`
 
 ### Throttling
-Rate limiting is enforced by the API with a threshold of 20 requests per second across all your organization's API keys, learn more about it [here](https://docs.onfleet.com/reference#throttling). 
+Rate limiting is enforced by the API with a threshold of 20 requests per second across all your organization's API keys, learn more about it [here](https://docs.onfleet.com/reference#throttling). We have implemented a limiter on the wrapper itself to avoid you from excessively hitting Onfleet's rate limitations and eventually be banned for.
```
Comment (dbhoot) [inline README.md]: I would change the copy to `unintentionally exceeding your rate limit`

Comment (dbhoot) [reply README.md]: obv, have to do the same for all languages

Comment (jamesliupenn) [reply README.md]: Language updated in https://github.com/onfleet/node-onfleet/pull/14/commits/753b8526032f72d1f80c2ad6e3930d78165a7984

**Comment by jamesliupenn** [comment]
@dbhoot let me know if the readme was the only thing you wanted to be changed. Please assign PR back to me so I can merge this and publish it on NPM. Thanks!

**Comment by dbhoot** [comment]
> @dbhoot let me know if the readme was the only thing you wanted to be changed. Please assign PR back to me so I can merge this and publish it on NPM. Thanks!

There's more testing I wanted to do but haven't yet. At a high level it looks good

**Comment by jamesliupenn** [comment]
> > @dbhoot let me know if the readme was the only thing you wanted to be changed. Please assign PR back to me so I can merge this and publish it on NPM. Thanks!
> 
> There's more testing I wanted to do but haven't yet. At a high level it looks good

No problem, just gonna hold it off until you do your tests! The more testing the merrier :)

**Pull request merged**